src/amo/components/Addon/index.js

```diff
@@ -154,22 +151,18 @@ export class AddonBase extends React.Component {
     this.props.toggleThemePreview(event.currentTarget);
   }
 
-  headerImage({ compatible }) {
+  headerImage() {
     const {
       addon,
       getBrowserThemeData,
-      i18n,
-      isPreviewingTheme,
-      installStatus,
     } = this.props;
     const previewURL = addon ? addon.previewURL : null;
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
     const iconUrl = getAddonIconUrl(addon);
 
     if (type === ADDON_TYPE_THEME) {
-      const label = isPreviewingTheme ? i18n.gettext('Cancel preview') : i18n.gettext('Tap to preview');
       const imageClassName = 'Addon-theme-header-image';
-      const headerImage = <img alt={label} className={imageClassName} src={previewURL} />;
+      const headerImage = <img alt="" className={imageClassName} src={previewURL} />;
```

Comment:
We should probably not leave this attribute empty. We could maybe add `"Preview of <addon.title>"`, what do you think?